instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import CommentSection from '../CommentSection/CommentSection';
+import PostHeader from './PostHeader'
+import PropTypes from "prop-types";
+import './post.css';
+
+const PostContainer = (props) => {
+    return (
+        <section className="post-container">
+            <section className="post-header">
+                <PostHeader thumbnail={props.thumbnail} username={props.username}/>
+            </section>
+            <section className="post">
+                <img src={props.image} alt="instagram" />
+            </section>
+            <section className="comment-container">
```

Comment:
does this mean you have two `section`s with the className of "comment-container"?